src/main.c

```diff
@@ -2408,15 +2409,85 @@ void handle_mouse_input() {
     }
 }
 
-void handle_movement(double dt) {
+/// [issue](https://github.com/Team-10-But-Better/Craft/issues/9)
+/// This function returns the movement speed based on whether the character should teleport.
+///
+///\param teleport : whether the character should teleport
+///\param teleportHeldLastFrame : whether the teleport button was previously held
+///\param groundedMovementSpeed : default movement speed
+
+int getGroundedMovementSpeed(bool teleport, bool *teleportHeldLastFrame, int groundedMovementSpeed)
+{
+    if (teleport)
```

Comment:
proper formatting of if and else if statement is required just like it is followed on line 2549.

Reply:
Fixed.